components/lib/dialog/Dialog.js

```diff
@@ -174,18 +174,20 @@ export const Dialog = React.forwardRef((inProps, ref) => {
             const leftPos = offset.left + deltaX;
             const topPos = offset.top + deltaY;
             const viewport = DomHandler.getViewport();
+            const leftMargin = parseFloat(getComputedStyle(dialogRef.current)['margin-left'])
+            const topMargin = parseFloat(getComputedStyle(dialogRef.current)['margin-top'])
```

Comment:
I think we need to look at ` const leftPos = offset.left + deltaX;` how the offset is calculated it should account for margin?